lint.sh

```diff
@@ -5,7 +5,8 @@ set -e
 FWDIR="$(cd "`dirname $0`"; pwd)"
 cd "$FWDIR"
 
-pycodestyle --max-line-length=100 --exclude mlflow/protos,mlflow/server/js,mlflow/store/db_migrations,mlflow/temporary_db_migrations_for_pre_1_users -- mlflow tests
+black --check --line-length=100 --exclude=mlflow/protos .
+pycodestyle --max-line-length=100   --ignore=E203,W503 --exclude mlflow/protos,mlflow/server/js,mlflow/store/db_migrations,mlflow/temporary_db_migrations_for_pre_1_users -- mlflow tests
```

Comment:
Ignore E203 and W503 because black violates these two rules: https://github.com/psf/black/blob/master/docs/compatible_configs.md#flake8